resources/views/faults/list.blade.php

```diff
@@ -0,0 +1,88 @@
+<div id="faults-table"></div>
+<script src="js/moment.min.js"></script>
+<script type="application/javascript">
+    $(document).ready(function () {
+        function status_translate(cell) {
+            if (typeof cell !== "string") {
+                cell = cell.getValue();
+            }
+            var translations = {
+                unseen: "@lang('faults.unseen')",
+                seen: "@lang('faults.seen')",
+                done: "@lang('faults.done')",
+                wont_fix: "@lang('faults.wont_fix')"
+            };
+            var colors = {
+                unseen: "text-info",
+                seen: "text-info",
+                done: "text-success",
+                wont_fix: "text-danger"
+            };
+            return '<div class="font-italic ' + colors[cell] + ' text-right">' + translations[cell] + '</div>';
+        }
+
+        var button = function (cell, formatterParams) {
```

Comment:
Maybe a reopen button could be added if something was closed by a mistake? Or if the reporters are not satisfied. I'm also ok if it will be done in a separate PR.

Reply:
That's a useful feature, I've added the reopen button.